lgc/patch/PatchPreparePipelineAbi.cpp

```diff
@@ -182,10 +182,14 @@ void PatchPreparePipelineAbi::setCallingConvs(Module &module) {
 // =====================================================================================================================
 // Set calling convention for the non-entry-points that do not yet have a calling convention set.
 //
+// Also change the calling convention of function definitions. The calling convention of the calls will be changed in
```

Comment:
You mean function _declarations_ (other than intrinsics and LGC internal calls).

I'm worried about changing the calls in PatchEntryPointMutate and changing the func decls here. If any calls are direct, then you have a window where the call does not match the func decl, and if InstCombine runs in that window the calls will disappear.